examples/heat-capacity/heat-capacity.py

```diff
@@ -231,15 +239,15 @@ def PrintResults(string,E,dE,Cv,dCv,types):
 
 for n in range(nBoots_work):
     if (n > 0):
-        print "Bootstrap: %d/%d" % (n,nBoots)
+        print("Bootstrap: %d/%d" % (n,nBoots))
     for k in range(K):
     # resample the results:
         if Nall_k[k] > 0:
             if (n == 0):  # don't randomize the first one
-                booti = numpy.array(range(N_k[k]))
+                booti = numpy.array(list(range(N_k[k])))
```

Comment:
Doesn't really need the `array(list(range(...` since the `array` will cast `range` correctly.